test/ci/buildkite/run_pipeline.sh

```diff
@@ -11,11 +11,19 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 PIPELINE_STEPS=" ${SCRIPT_DIR}/setup.yml "
 if [ "${BUILDKITE_PIPELINE_NAME}" == "ngtf-cpu-ubuntu-grappler" ]; then
    export BUILD_OPTIONS=--use_grappler
+   export TF_LOCATION=/localdisk/buildkite-agent/prebuilt_tensorflow_2_2_0
```

Comment:
Can we move the TF_LOCATION line as a common to top?

Reply:
No, because it is different for Intel TF